### PR TITLE
[CARBONDATA-2106] Update product document with page level reader property

### PR DESCRIPTION
**Pull request by vinayakumarb**
Updated the document.

### CODE REVIEW

**Comment by ravipesala** [comment]
Can one of the admins verify this patch?

**Comment by sgururajshetty** [comment]
LGTM

File: docs/configuration-parameters.md [hunk 1]
```diff
@@ -119,6 +119,7 @@ This section provides the details of all the configurations required for CarbonD
 | carbon.numberof.preserve.segments | 0 | If the user wants to preserve some number of segments from being compacted then he can set this property. Example: carbon.numberof.preserve.segments = 2 then 2 latest segments will always be excluded from the compaction. No segments will be preserved by default. |
 | carbon.allowed.compaction.days | 0 | Compaction will merge the segments which are loaded with in the specific number of days configured. Example: If the configuration is 2, then the segments which are loaded in the time frame of 2 days only will get merged. Segments which are loaded 2 days apart will not be merged. This is disabled by default. |
 | carbon.enable.auto.load.merge | false | To enable compaction while data loading. |
+|carbon.enable.page.level.reader.in.compaction|true|Enabling page level reader for compaction reduces the memory usage while compacting more number of segments. It allows reading only page by page instead of reaing whole blocklet to memory.|
```
Comment (ravipesala) [inline docs/configuration-parameters.md]: typo `reading` not `reaing`

Comment (vinayakumarb) [reply docs/configuration-parameters.md]: Fixed

**Comment by ravipesala** [comment]
LGTM

**Pull request closed without merging**